aksel.nav.no/website/sanity/plugins/publication-flow/actions/approve.tsx

```diff
@@ -15,11 +17,16 @@ export const createWrappedApproveAction = () => {
   ): DocumentActionDescription | null => {
     const { patch, publish } = useDocumentOperation(props.id, props.type);
     const [dialogOpen, setDialogOpen] = useState(false);
-    const lastVerified = props.published?.updateInfo?.["lastVerified"];
+
     if (!props.published) {
       return null;
     }
 
+    const verifiedDocument = props.published as SanityDocument &
+      Oppdateringsvarsel;
```

Comment:
Did you forget to add `| null` here (as in focusAction.tsx), or do you know that `published` will be defined here?

Reply:
Fixed 🐛